### ISSUE TITLE
error while starting up

### ISSUE THREAD

**Issue by akkartik** [arclanguage/anarki#62]
Hi Ross, have you been running arc using `arc.sh`? It isn't working for me on OS X.

```bash
$ ./arc.sh
lifted.0.0: undefined;
 cannot reference an identifier before its definition
  in module: ac.scm
  context...:
   ac.scm:1295:0: aload1
   boot.scm: [running body]
```

The error message isn't very useful, but I did a binary search and narrowed it down to [this line](https://github.com/arclanguage/anarki/commit/29468035e57f7e711d94622e7e4b8ffc1126f51d#diff-0e7ba2eab5b8bb541b9c8226bdd0f838R1111) in `ac.scm`:

```
(require racket/random)
```

But I'm not sure how to fix it. Any ideas? Thanks.

**Comment by akkartik** [comment]
Turns out my racket v6.3 (the default on Homebrew for OS X El Capitan) is not recent enough. It works with v6.9. I don't know about intermediate versions. We should update the documentation.

**Comment by rocketnia** [comment]
Thanks for figuring out what was going on there!

I didn't do very diligent testing before I committed my Windows fixes. I didn't test on older versions of Racket, and I didn't test on anything but one Windows 10 machine either (at least until after my commit went to GitHub, at which point Travis CI ran the unit tests on another machine for me).

That require line is something I added as part of the Windows fixes. It let me change news.arc to not read from /dev/urandom on Windows. I wasn't sure if this would break on old versions of Racket, and I guess it does. Sorry I cost you that time tracking it down.
